File: 3334-apple-redistribution-into-boxes/apple-redistribution-into-boxes.java
class Solution {
    public int minimumBoxes(int[] apple, int[] capacity) {
        int ans=0;
        int val= 0,sum=0;
        for(int i : apple){
            sum+=i;
        }
        
        Arrays.sort(capacity);

        for (int i= capacity.length-1;i>=0;i--){
            val+=capacity[i];
            if(val>=sum){
                ans++;
                return ans;
            }else ans++;
        }
        return ans;
    }
}

File: 3334-apple-redistribution-into-boxes/apple-redistribution-into-boxes.py
class Solution:
    def minimumBoxes(self, apple: List[int], capacity: List[int]) -> int:
        ans=0
        maxlen=1
        val = 0

        capacity.sort(reverse=True)
        tota_appel=sum(apple)

        for i in capacity:
            val+=i
            if val >= tota_appel:
                ans+=1
                return ans
            else:
                ans+=1


        